### PR TITLE
Converted Redstone, Server Ping and Item moves in inventory events.  Updated POM to handle Vault 1.5.4.

### PR DESCRIPTION
**Pull request by Talamar1**

### CODE REVIEW

**Comment by mcmonkey4eva** [comment]
In the future, try to keep it one commit per event update.

File: pom.xml [hunk 1]
```diff
@@ -34,10 +34,7 @@
             <id>everything</id>
             <url>http://repo.citizensnpcs.co</url>
         </repository>
-        <repository>
-            <id>comphenix-repo</id>
-            <url>http://repo.comphenix.net/content/groups/public</url>
-        </repository>
```
Comment (mcmonkey4eva) [inline pom.xml]: Why is this removed?!

File: pom.xml [hunk 2]
```diff
@@ -84,8 +81,7 @@
         <dependency>
             <groupId>net.milkbowl.vault</groupId>
             <artifactId>Vault</artifactId>
-            <version>1.4.2-SNAPSHOT</version>
-            <scope>provided</scope>
+            <version>1.5.4</version>
```
Comment (mcmonkey4eva) [inline pom.xml]: Eh... where'd the scope go?

File: src/main/java/net/aufdemrand/denizen/events/scriptevents/ItemMoveScriptEvent.java [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+                || lower.equals("item moves from inventory to " + destinationType);
+    }
+
+    @Override
+    public String getName() {
+        return "ItemMoves";
+    }
+
+    @Override
+    public void init() { Bukkit.getServer().getPluginManager().registerEvents(this, DenizenAPI.getCurrentInstance()); }
```
Comment (mcmonkey4eva) [inline src/main/java/net/aufdemrand/denizen/events/scriptevents/ItemMoveScriptEvent.java]: Don't try to one-line shit please

File: src/main/java/net/aufdemrand/denizen/events/scriptevents/ItemMoveScriptEvent.java [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+        item = new dItem(event.getItem());
+        itemSet = false;
+        cancelled = event.isCancelled();
+        fire();
+        event.setCancelled(cancelled);
+        if (itemSet) {
+            event.setItem(item.getItemStack());
+        }
+    }
+}
```
Comment (mcmonkey4eva) [inline src/main/java/net/aufdemrand/denizen/events/scriptevents/ItemMoveScriptEvent.java]: newline at end of file

File: src/main/java/net/aufdemrand/denizen/events/scriptevents/ListPingScriptEvent.java [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+        return true;
+    }
+
+    @Override
+    public String getName() {
+        return "ListPing";
+    }
+
+    @Override
+    public void init() { Bukkit.getServer().getPluginManager().registerEvents(this, DenizenAPI.getCurrentInstance()); }
```
Comment (mcmonkey4eva) [inline src/main/java/net/aufdemrand/denizen/events/scriptevents/ListPingScriptEvent.java]: one lining bad

File: src/main/java/net/aufdemrand/denizen/events/scriptevents/RedstoneScriptEvent.java [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+        return true;
+    }
+
+    @Override
+    public String getName() {
+        return "RedstoneRecalculated";
+    }
+
+    @Override
+    public void init() { Bukkit.getServer().getPluginManager().registerEvents(this, DenizenAPI.getCurrentInstance()); }
```
Comment (mcmonkey4eva) [inline src/main/java/net/aufdemrand/denizen/events/scriptevents/RedstoneScriptEvent.java]: One liiiinniiiiiiiiinnnnnnngggggggg

**Comment by mcmonkey4eva** [comment]
@Morphan1 could you please look this over and leave comments?

File: pom.xml (hunk 2 above)
Comment (Talamar1) [inline pom.xml]: I used the xml from their site, without the scope, it seems to download.

File: pom.xml (hunk 1 above)
Comment (Talamar1) [inline pom.xml]: Because Protocollib isn't needed anymore, and that whole site is down.

File: pom.xml (hunk 2 above)
Comment (mcmonkey4eva) [inline pom.xml]: Scope is related to how it's stored /after/ downloading, it has nothing to do with whether it downloads properly.
